Resources/Storage/DIPStorage.py

```diff
@@ -106,8 +106,14 @@ def __putFile( self, src_file, dest_url ):
     localCache = False
     srcDict = res['Value']
     if srcDict['Protocol'] in ['dips', 'dip']:
+      serviceDict = dict( srcDict )
```

Comment:
Can you please add comments ? I don't really understand, in particular the ```[:3]```, nor why FileName is empty. Does that effectively make dips a third party protocol ?